apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/v1/controller/ItemController.java

```diff
@@ -60,6 +60,10 @@ public OpenItemDTO createItem(@PathVariable String appId, @PathVariable String e
       throw new BadRequestException("User " + item.getDataChangeCreatedBy() + " doesn't exist!");
     }
 
+    if( (item.getComment() == null ? 0 : item.getComment().length())  > 64){
```

Comment:
`Objects.isnull(...)` would be more graceful...

Reply:
yeah